home/.config/home-manager/user-github-actions.nix

```diff
@@ -0,0 +1,6 @@
+{ ... }:
+
+{
+  imports = [ ./home.nix ];
+  home.username = "runner";
+}
```

Comment:
This file should be put outsides the `home/`, but actually not installed into machines, so acceptable for now... 😩